api/modelmanager/modelinfo_test.go

```diff
@@ -31,25 +32,42 @@ func (s *modelInfoSuite) assertResponse(c *gc.C, result interface{}) *params.Mod
 	return result.(*params.ModelInfoResults)
 }
 
-func (s *modelInfoSuite) TestModelInfo(c *gc.C) {
-	results := params.ModelInfoResults{
-		Results: []params.ModelInfoResult{{
-			Result: &params.ModelInfo{Name: "name", UUID: "etc."},
-		}, {
-			Error: &params.Error{Message: "woop"},
-		}},
-	}
+func (s *modelInfoSuite) assertExpectedModelInfo(c *gc.C, expectedInfo params.ModelInfoResults) {
 	apiCaller := basetesting.APICallerFunc(
 		func(objType string, version int, id, request string, a, result interface{}) error {
 			s.checkCall(c, objType, id, request)
 			resp := s.assertResponse(c, result)
-			*resp = results
+			*resp = expectedInfo
```

Comment:
So this function is injecting the expected response and then asserting that it got the expected response back? What piece is actually being tested?